lib/msal-common/src/client/SilentFlowClient.ts

```diff
@@ -15,10 +15,11 @@ import { ClientConfigurationError } from "../error/ClientConfigurationError";
 import { ResponseHandler } from "../response/ResponseHandler";
 import { CacheRecord } from "../cache/entities/CacheRecord";
 import { CacheOutcome } from "../utils/Constants";
+import { IPerformanceClient } from "../telemetry/performance/IPerformanceClient";
 
 export class SilentFlowClient extends BaseClient {
-
-    constructor(configuration: ClientConfiguration) {
+    
+    constructor(configuration: ClientConfiguration, performanceClient?: IPerformanceClient) {
         super(configuration);
```

Comment:
Either pass through `performanceClient` to `super` or remove the constructor entirely (technically superfluous here).

Reply:
fixed